### PR TITLE
feat: Add ability to go back and modify entry type fields

### LINKED ISSUE #53: feat: Add ability to go back when adding entries
I had the experience of realizing I made a mistake while adding a field, and had to ctrl-c out and start over.   It would be really nice to be able to navigate through the fields you have created, maybe with a list of the created fields so far, or something like that.

### PR DESCRIPTION
**Pull request by c12i**
This PR will close #53

### CODE REVIEW

**Comment by jost-s** [comment]
I always get an error that a flake.nix is missing:
```sh
$ hc-scaffold example forum
✔ Choose UI framework: (Use arrow-keys. Return to submit) · lit
✔ Choose a package manager: (Use arrow-keys. Return to submit) · npm
Setting up nix development environment...
error: path '/nix/store/90xdfxqppbfn7zk92z0hnjf9j5mfsham-source/forum/flake.nix' does not exist
Error setting up the nix environment
```
It happens regardless of template and with `hello-world` too. What is missing?

File: src/scaffold/entry_type/fields.rs [hunk 1]
```diff
@@ -45,25 +46,95 @@ pub fn choose_fields(
...
+                        zome_file_tree,
+                        field_types_templates,
+                        no_ui,
+                        None,
+                    )?;
+                    fields.push(new_field);
+                }
+                2 => {
+                    // Remove field
+                    if !fields.is_empty() {
```
[the remark this answers is not in the document]
Comment (c12i) [reply src/scaffold/entry_type/fields.rs:110]: Good idea, added

**Comment by c12i** [comment]
@jost-s this is strange, Ia m unable to reproduce, were you running this inside or outside a nix shell?

**Comment by jost-s** [comment]
I was running it outside of a nix shell. Now running inside of it, I get the same error:
```sh
nix: scaffolding (feat/entry-type-go-back) $ hc-scaffold example hello-world
✔ Choose a package manager: (Use arrow-keys. Return to submit) · npm
Setting up nix development environment...
error: path '/nix/store/90xdfxqppbfn7zk92z0hnjf9j5mfsham-source/hello-world/flake.nix' does not exist
Error setting up the nix environment 
```

I'll start from scratch again.

**Comment by jost-s** [comment]
I end up at the same error. Here's what I do:
```
git clone ...
cd scaffolding
cargo install --path .
nix develop
hc-scaffold example hello-world
confirm default npm
```

**Comment by c12i** [comment]
Interesting, I not able to reproduce. Are you running this inside or outside a nix shell

**Comment by matthme** [comment]
But I can reproduce it if I run it inside the scaffolding repo.

**Review by matthme — APPROVED** [review]
It works for me and looks great!

**Comment by jost-s** [comment]
Right, that's the difference then. I was able to test like this before, I must have done it in a separate folder. Thanks @matthme

**Pull request merged**